WordPress/Classes/Utility/WPUserAgent.h

```diff
@@ -6,25 +6,16 @@
  */
 @interface WPUserAgent : NSObject
 
-#pragma mark - Changing the user agent
-
 /**
- *  @brief      Sets the App's user agent to be the default one.
+ *  @brief      Sets User-Agent header of all UIWebViews to be the WordPress one.
  */
-- (void)useDefaultUserAgent;
-
-/**
- *  @brief      Sets the App's user agent to be the WordPress one.
- */
-- (void)useWordPressUserAgent;
-
-#pragma mark - Getting the user agent
+- (void)useWordPressUserAgentInUIWebViews;
```

Comment:
The method was called `useWordPressUserAgent` previously, but I've decided to rename it to `useWordPressUserAgentInUIWebViews` in order to clarify what does the method actually do - it sets default UA for all UIWebView instances via NSUserDefaults.

Other HTTP users (e.g. NSURLRequest) still have to set UA manually themselves by calling `wordPressUserAgent` because NSUserDefaults UA value gets read only by UIWebViews.